doc/Installation/Installation-CentOS-7-Nginx.md

```diff
@@ -0,0 +1,144 @@
+
+### DB Server ###
+
+> NOTE: Whilst we are working on ensuring LibreNMS is compatible with MySQL strict mode, for now, please disable this after mysql is installed.
+
+#### Install / Configure MySQL
+```bash
+yum install mariadb-server mariadb
+mysql -uroot -p
+```
+
+```sql
+CREATE DATABASE librenms;
+GRANT ALL PRIVILEGES ON librenms.*
+  TO 'librenms'@'localhost'
+  IDENTIFIED BY '<password>'
+;
+FLUSH PRIVILEGES;
+exit
+```
+
+`vim /etc/my.cnf`
+
+Within the [mysqld] section please add:
+
+```bash
+innodb_file_per_table=1
+sql-mode=""
+```
+
+```service mysql restart```
+
+### Web Server ###
+
+#### Install / Configure Nginx 
+
+`yum install epel-release`
+
+`yum install php70w php70w-cli php70w-gd php70w-mysql php70w-snmp php70w-pear php70w-curl php70w-common php70w-fpm nginx net-snmp mariadb ImageMagick jwhois nmap mtr rrdtool MySQL-python net-snmp-utils cronie php70w-mcrypt fping git`
+
+In `/etc/php.ini` ensure date.timezone is set to your preferred time zone.  See http://php.net/manual/en/timezones.php for a list of supported timezones.  Valid examples are: "America/New York", "Australia/Brisbane", "Etc/UTC".
+
+```bash
+service php7.0-fpm restart
+phpenmod mcrypt
+```
+
+#### Add librenms user
+
+```bash
+useradd librenms -d /opt/librenms -M -r
+usermod -a -G librenms apache
+```
+
+#### Clone repo
+
+```bash
+cd /opt
+git clone https://github.com/librenms/librenms.git librenms
+```
+
+#### Web interface
+
+```bash
+cd /opt/librenms
+mkdir rrd logs
+chmod 775 rrd
+vim /etc/nginx/conf.d/librenms.conf
+```
+
+Add the following config:
+
+```nginx
+server {
+ listen      80;
+ server_name librenms.example.com;
+ root        /opt/librenms/html;
+ index       index.php;
+ access_log  /opt/librenms/logs/access_log;
+ error_log   /opt/librenms/logs/error_log;
+ location / {
+  try_files $uri $uri/ @librenms;
+ }
+ location ~ \.php {
+  include fastcgi.conf;
+  fastcgi_split_path_info ^(.+\.php)(/.+)$;
+  fastcgi_pass unix:/var/run/php/php7.0-fpm.sock;
+ }
+ location ~ /\.ht {
+  deny all;
+ }
+ location @librenms {
+  rewrite api/v0(.*)$ /api_v0.php/$1 last;
+  rewrite ^(.+)$ /index.php/$1 last;
+ }
+}
+```
+
+```bash
+service nginx restart
+```
+
+#### Web installer
+
+Now head to: http://librenms.example.com/install.php and follow the on-screen instructions.
+
+#### Configure snmpd
+
+```bash
+cp /opt/librenms/snmpd.conf.example /etc/snmpd.conf
+vim /etc/snmpd.conf
+```
+
+Edit the text which says `RANDOMSTRINGGOESHERE` and set your own community string.
+
+`service snmpd restart`
+
+#### Cron job
+
+`cp librenms.nonroot.cron /etc/cron.d/librenms`
+
+#### Final steps
+
+```bash
+chown -R librenms:librenms /opt/librenms
+```
+
+Now run validate your install and make sure everything is ok:
+
+```bash
+cd /opt/librenms
+./validate.php
```

Comment:
My install needed the path to fping before being able to pass the validate.php check.